app/helpers/tickets_helper.rb

```diff
@@ -1,13 +1,9 @@
 module TicketsHelper
   # Defines all Ticket types to select from
   @@ticket_types = ['Open Source', 'Competitive', 'ML', 'PClub Activities', 'Other']
-
-  def status_word(status_bool)
-    if status_bool
-      "Approved"
-    else
-      "Pending"
-    end
+  @@ticket_status = {:pending => "Pending", :approved => "Approved", :deleted => "Deleted"}
+  def status_word(status)
+    @@ticket_status[status]
```

Comment:
Unfortunately this does not work, since when we actually use it, in the ticket rendering view, ticket.status turns out to be a string. So, this could be replaced by `status.titleize` or something similar...